Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -488,7 +491,7 @@ extension BrowserViewController: WKNavigationDelegate {
         // gives us the exact same behaviour as Safari.
         if ["sms", "tel", "facetime", "facetime-audio"].contains(url.scheme) {
             if url.scheme == "sms" { // All the other types show a native prompt
-                showSnackbar(forExternalUrl: url, tab: tab) { isOk in
+                presentExternalLinkAlert(url) { isOk in
```

Comment:
Needing `// Ecosia` comment? Not a strong opinion on this. As the function won't exist anymore 😅

Reply:
I added it above the `presentExternalLinkAlert` method definition (which used to be `showSnackbar`) and thought it was already enough to give context to the places where it is used as well